### ISSUE TITLE
fix: `FluentProfileMenu`'s inner `FluentPersona` component does not show initials

### ISSUE THREAD

**Issue by thebarrettlo** [microsoft/fluentui-blazor#2286]
# 🐛 Bug Report

When using the default implementation of `FluentProfileMenu` with the `FullName` property set but not `Image`, the "outer" `FluentPersona` displays the initials from the `FullName` property. However, the inner `FluentPersona` in the popover does not display initials, only a blank, colored circle.

## 💻 Repro or Code Sample

```razor
<FluentProfileMenu FullName="First Last" />
```

e.g.,

![image](https://github.com/microsoft/fluentui-blazor/assets/36118918/ed95c3f2-12eb-4326-ab0e-e95bfca71937)

## 🤔 Expected Behavior

The "outer" `FluentPersona` image/initials matches the one in the popover.

## 😯 Current Behavior

When an image is not set on `FluentProfileMenu`, the inner `FluentPersona` in the popover does not display initials, only a blank, colored circle.

## 💁 Possible Solution

[On the `FluentPersona` in the popover](https://github.com/microsoft/fluentui-blazor/blob/6072e948565902a59ffe56da8c1fad21530cc1d0/src/Core/Components/ProfileMenu/FluentProfileMenu.razor#L49), set the `Name` property to `FluentProfileMenu`'s `FullName` property so that the initials can be parsed.

## 🌍 Your Environment

* OS & Device: Windows 11 on PC
* Browser: Microsoft Edge
* .NET 8, Fluent UI Blazor library 4.8.0

**Comment by dvoituron** [comment]
Will be fixed: See #2294

In the meantime, you can use the `Initials` attribute to set this value manually.
```xml
<FluentProfileMenu FullName="First Last" Initials="FL" />
```

**Comment by thebarrettlo** [comment]
Thank you @dvoituron!
